extraction/README.md

```diff
@@ -0,0 +1,18 @@
+# Extraction Demo
+
+Code referenced from the [manga109api](https://github.com/manga109/manga109api) repository.
+
+## Usage
+Executing `python extract.py` will output a cropped image/s based on the specified annotation provided.
+
+When using the code, please make sure this script is in the manga109api directory along with the dataset. For a usage example, if you want to extract a faces from 5 pages of YumeNoKayoiji's book, please run the following command : 
+
+```python
```

Comment:
Just a minor thing. This should be ` bash` or `console`, not `python`

Reply:
@stevengans Could you change this to `bash` too?

Reply:
@matsui528 can you clarify the ask. Do you want this as a .sh file instead of a .py file?

Reply:
I think @matsui528 's commenting on the
```
 ```python
 python extract.py --book YumeNoKayoiji --annotation face --pages 5 --preprocess --size 256
```
part. Since this is executed in a shell like bash, for proper syntax highlighting, the
```
 ```python
```
should be changed to:
```
 ```bash
```

Reply:
Done

Reply:
Thanks!